src/test/integration/suite/smoke/module.smoke.ts

```diff
@@ -0,0 +1,106 @@
+import { expect } from 'chai';
+import { EditorView, VSBrowser, WebDriver } from 'vscode-extension-tester';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as tmp from 'tmp';
+import { Project } from '../../util/project';
+import { dismissNotifications, notificationExists } from '../../util/common';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/test/integrat...
  
  
  DangerID: danger-id-Danger;
  File: src/test/integration/suite/smoke/module.smoke.ts;
  Line: 7;
-->

- :warning: src/test/integration/suite/smoke/module.smoke.ts line 7 – 'notificationExists' is defined but never used. (@typescript-eslint/no-unused-vars)